apistructs/job.go

```diff
@@ -115,6 +115,26 @@ type JobFromUser struct {
 	ContainerInstanceProvider *ContainerInstanceProvider `json:"containerInstanceProvider,omitempty"` // ContainerInstanceProvider pointer type if not hitted of specified, it will be nil
 }
 
+func (job *JobFromUser) GetOrgName() string {
+	if job.Labels != nil {
+		orgName, ok := job.Labels["DICE_ORG_NAME"]
```

Comment:
use DICE_ORG_NAME & DICE_ORG_ID in apistructs.

Reply:
adjusted, and use apistructs define in prepare
![image](https://user-images.githubusercontent.com/30427474/163779651-682f24ad-b83c-4a45-8381-011f535bf81c.png)